pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["setuptools >= 40.9.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+src_paths = ["lsqecc", "test"]
+
+[tool.black]
+line-length = 100
+
+[tool.mypy]
+ignore_missing_imports = True
+exclude = debug/ 
```

Comment:
Wonder why this looks weird. Also I do hope to see `debug/` gone, but I don't think we are there yet

Reply:
Just fixed it. I think testing is implemented we will look to remove `debug` (maybe with some jupyter notebooks to demonstrate some example use cases of the tool).

Reply:
Yeah that's a great idea ... have some notebooks like they do in qiskit